src/main/java/org/jboss/logmanager/handlers/WriterHandler.java

```diff
@@ -19,10 +19,10 @@
 
 package org.jboss.logmanager.handlers;
 
-import java.io.BufferedWriter;
-import java.io.Closeable;
-import java.io.Flushable;
-import java.io.Writer;
+import java.io.*;
```

Comment:
No start imports please

Reply:
Oops, I realized this is not the Quarkus repo, so forget about it :)

Reply:
It's okay, I don't want star imports here either :)